consensus/state_sync/helpers.go

```diff
@@ -6,32 +6,25 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
-func (m *stateSync) SendStateSyncMessage(stateSyncMsg *typesCons.StateSyncMessage, peerAddress cryptoPocket.Address, height uint64) error {
-	anyMsg, err := anypb.New(stateSyncMsg)
+// SendStateSyncMessage sends a state sync message after converting to any proto, to the given peer
+func (m *stateSync) SendStateSyncMessage(msg *typesCons.StateSyncMessage, dst cryptoPocket.Address, height uint64) error {
+	anyMsg, err := anypb.New(msg)
 	if err != nil {
 		return err
 	}
-
-	m.logger.Info().Fields(m.logHelper(peerAddress.ToString())).Msg("Sending StateSync Message")
-	return m.sendToPeer(anyMsg, peerAddress)
-}
-
-// Helper function for messages to the peers
-func (m *stateSync) sendToPeer(msg *anypb.Any, peerAddress cryptoPocket.Address) error {
-	if err := m.GetBus().GetP2PModule().Send(peerAddress, msg); err != nil {
-		m.logger.Error().Msgf(typesCons.ErrSendMessage.Error(), err)
+	if err := m.GetBus().GetP2PModule().Send(dst, anyMsg); err != nil {
+		m.logger.Error().Err(err).Msg(typesCons.ErrSendMessage.Error())
 		return err
 	}
 	return nil
 }
 
-func (m *stateSync) logHelper(receiverPeerAddress string) map[string]any {
+func (m *stateSync) StateSyncLogHelper(receiverPeerAddress string) map[string]any {
```

Comment:
A helper (that's not in the shared repo) should NEVER be publically exposed. If it is, it means we're doing something wrong.